scripts/export-import-test.py

```diff
@@ -0,0 +1,121 @@
+import sys
+import argparse
+import subprocess
+import os
+import shutil
+
+
+def getVersion(executablePath):
+    try:
+        result = subprocess.run(
+            [executablePath, "--version"],
+            capture_output=True,
+            text=True,
+            check=True
+        )
+        output = result.stdout.strip()
+        if output.startswith("Kuzu "):
+            return output.split(" ", 1)[1]
+        else:
+            print("Unexpected version format:", output)
+            return None
+    except subprocess.CalledProcessError as e:
+        print(f"Error running '{executablePath} --version': {e}")
+        return None
+
+
+def runCommand(cmd, cwd=None, env=None, capture_output=False):
+    if isinstance(cmd, str):
+        cmd = cmd.split()
+
+    print(f"> Running: {' '.join(cmd)} (cwd={cwd})")
+
+    if capture_output:
+        result = subprocess.run(cmd, cwd=cwd, env=env, text=True, capture_output=True, check=True)
+        print(result.stdout)
+        return result.stdout.strip()
+    else:
+        process = subprocess.Popen(
+            cmd,
+            cwd=cwd,
+            env=env,
+            text=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            bufsize=1,
+            universal_newlines=True,
+            stdin=subprocess.DEVNULL
+        )
+
+        for line in process.stdout:
+            print(line, end='')
+
+        process.stdout.close()
+        retcode = process.wait()
+        if retcode != 0:
+            raise subprocess.CalledProcessError(retcode, cmd)
+        return None
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Export DBs from datasetDir using commitA and test in commitB"
+    )
+    parser.add_argument("commitA", help="Git commit to export databases from")
+    parser.add_argument("commitB", help="Git commit to test against")
+    parser.add_argument("datasetDir", help="Path to the dataset directory")
+    args = parser.parse_args()
+
+    commitA = args.commitA
+    commitB = args.commitB
+    datasetDir = args.datasetDir
+
+    scriptDir = os.path.dirname(os.path.realpath(__file__))
+    kuzuRoot = os.path.abspath(os.path.join(scriptDir, ".."))
+    currentBranch = runCommand(["git", "rev-parse", "--abbrev-ref", "HEAD"], cwd=kuzuRoot, capture_output=True)
+
+    try:
+        # Checkout commit A and build
+        runCommand(["git", "checkout", commitA], cwd=kuzuRoot)
+        runCommand(["make", "extension-build", "EXTENSION_LIST=json"], cwd=kuzuRoot)
+
+        # Switch back to working branch (to use the following script)
+        runCommand(["git", "checkout", currentBranch], cwd=kuzuRoot)
+
+        # Export databases
+        exportScriptPath = os.path.join(kuzuRoot, "scripts", "export-dbs.py")
+        execPath = os.path.join(kuzuRoot, "build", "relwithdebinfo", "tools", "shell", "kuzu")
+        runCommand(["python3", exportScriptPath, execPath, datasetDir], cwd=kuzuRoot)
```

Comment:
Set the export path to `--output-dir / base-commit-version`.
Also check if this directory is already present and skip exporting if exists.

To make it robust, here's the workflow I imagine happening:
1. Check if `--output-dir / base-commit` exists
  a. If it exists, do nothing
  b. If it does not exist, checkout base commit and run export for `--output-dir / base-commit-version_inprogress`. Perhaps also remove any existing path.
c. If export completes successfully, rename `--output-dir / base-commit-version_inprogress` to `--output-dir / base-commit-version`. This is to make the export **atomic**.
2. Now run the tests